docs/guides/use-shared-project-for-organization-wide-collaboration.md

```diff
@@ -9,81 +9,83 @@ import BulkAddUserImg from './img/invite-all-organization-members.png';
 
 <span class="badge badge--cloud">Camunda 8 only</span>
 
-Organizations often manage process documentation across multiple layers, from high-level strategic models to detailed operational workflows. [Process landscape visualization](/docs/components/modeler/web-modeler/process-landscape-visualization.md) streamlines this complexity by supporting different levels of abstraction, enabling seamless collaboration across teams, and providing tailored insights for various stakeholders.
+Organizations often manage process documentation across multiple layers, from high-level strategic models to detailed operational workflows. [Process landscape visualization](/components/modeler/web-modeler/process-landscape-visualization.md) streamlines this complexity by supporting different levels of abstraction, enabling seamless collaboration across teams, and providing tailored insights for various stakeholders.
 
 ## About this guide
 
 This guide shows how you can consolidate these perspectives into a single shared project, ensuring clarity, reusability, and governance within your organization’s efforts.
 
 Teams can use shared projects to:
 
-- Prevent duplicate work.
-- Maintain a single source of truth.
-- Enable cross-project navigation and transparency.
-- Govern access to organization resources effectively.
+- Prevent duplicate work
+- Maintain a single source of truth
+- Enable cross-project navigation and transparency
+- Govern access to organization resources effectively
 
-## Create a shared project
+## Create a shared project and invite collaborators
 
-To create a shared project as a repository for your organization's resources:
+If you have not yet already, [create a Camunda 8 account](/guides/create-account.md)
 
-Open Modeler, and select **Create new project** to create a new project to store resources in.
+To create a shared project as a space for your organization's resources:
 
-![web modeler empty home](../components/modeler/web-modeler/img/web-modeler-new-user-home.png)
+1. Open [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md) and select **Create new project** (if you do not have any existing projects) or **New project** (if you already have existing projects).
+2. Name your project accordingly.
+   ![web modeler empty home](../components/modeler/web-modeler/img/web-modeler-new-user-home.png)
+3. On the right side of the page under **Collaborators**, select **Add user**.
+4. Enter the email address of the collaborator you would like to invite and select `Enter` or `Tab` between each entry. You may enter up to 20 email addresses. You can also use the **All users in the organization** option to send a bulk invitation to all colleagues.
```

Comment:
There are 2 cases:
1. I want to add colllaborators to my organization AND my project - enter the email addresses
2. I want to add all existing organization members to my project - use the bulk invite